KPP/OHreact_parser.py

```diff
@@ -1,154 +1,213 @@
 #!/usr/bin/env python
-#
-# Helper routine to generate a Fortran code snipped that computes the OH
-# reactivity based on the reactions listed in gckpp_Monitor.F90.
-# The OH reactivity is defined as the inverse of its lifetime and is
-# determined using the reaction rates of all reactions that consume OH.
-# The generated code snipped can then be inserted into one of the Fortran
-# modules, e.g. gckpp_Util.F90.
-#
-# Usage:
-# python OHreact_parser.py
-#
-# Revision History:
-# 2018-06-18 - christoph.a.keller@nasa.gov - initial version
-# See git history for subsequent updates
-
-# Starts here
+"""
```

Comment:
I reorganized this so that we can set OH reactivity = 0 for the carboncycle mechanism.